app/views/katello/api/v2/environments/show.json.rabl

```diff
@@ -6,6 +6,16 @@ extends 'katello/api/v2/common/org_reference'
 attributes :library
 attributes :registry_name_pattern, :registry_unauthenticated_pull
 
+if @content_source.present?
+  node :content_source do |env|
+    {
+      :name => @content_source.name,
+      :id => @content_source_id,
+      :environment_is_associated => env.content_source_associated?(@content_source)
+    }
+  end
+end
```

Comment:
This doesn't seem to work. Content source for an env is supposed to be an array given multiple smart proxies can have multiple environments. Hence will need to change this here..